src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,86 @@
-package core.basesyntax;
-
-public class MyHashMap<K, V> implements MyMap<K, V> {
-
-    @Override
-    public void put(K key, V value) {
-
-    }
-
-    @Override
-    public V getValue(K key) {
-        return null;
-    }
-
-    @Override
-    public int getSize() {
-        return 0;
-    }
-}
+package core.basesyntax;
+
+public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final double LOAD_FACTOR = 0.75;
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int GROW_COEFFICIENT = 2;
+    private int treshold = (int) (DEFAULT_CAPACITY * LOAD_FACTOR);
+    private Node<K, V>[] table = new Node[DEFAULT_CAPACITY];
+    private int capacity = DEFAULT_CAPACITY;

```

Comment:
let's initialize the variables in the constructor and do you really need the capacity variable?